debian/changelog

```diff
@@ -8,7 +8,7 @@ cuttlefish-common (0.9.11) UNRELEASED; urgency=medium
 
  -- Jason Macnak <natsu@google.com> Tue, 11 Feb 2020 14:56:24 -0800
 
-cuttlefish-common (0.9.10) UNRELEASED; urgency=medium
+cuttlefish-common (0.9.12) stable; urgency=medium
```

Comment:
This seems like rewriting history a bit, don't understand the reason for this change.

Reply:
Correct, the goal of this change is to sync the version numbers in this changelog with the released images.